packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/utils/getDisplayedFilters.js

```diff
@@ -13,6 +13,26 @@ const customOperators = [
 ];
 
 const getDisplayedFilters = ({ formatMessage, users }) => {
+  const getDisplaynameFromUser = (user) => {
+    if (user.username) {
+      return user.username;
+    }
+    if (user.firstname && user.lastname) {
```

Comment:
Is there a reason we cant rely on one or the other? Do they need to be both set?